doc/api/process.md

```diff
@@ -1920,10 +1944,12 @@ cases:
 [`process.exitCode`]: #process_process_exitcode
 [`process.kill()`]: #process_process_kill_pid_signal
 [`promise.catch()`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch
+[`process.on('uncaughtException')`]: process.html#process_event_uncaughtexception
```

Comment:
A nit: out of ABC order)

Reply:
Mh - it's o after c, right?

Reply:
The one before it is `promise` not `process`. Tripped me up the first few times I read it.

Reply:
`process.kill`
`process.on`
`promise.catch` ?)

Reply:
Ah, yup. They look so similar …